client/extensions/updateArticleOnProfileChange/src/extension.ts

```diff
@@ -19,9 +19,9 @@ const extension: IExtension = {
                             superdesk.entities.contentProfile.get(current.profile),
                             superdesk.entities.contentProfile.get(next.profile),
                         ]).then((res) => {
-                            const [currentProfile, nextProfile] = res;
+                            const nextProfile = res[1];
 
-                            if (currentProfile.label === 'ALERT' && nextProfile.label !== 'ALERT') {
+                            if (nextProfile.label !== 'ALERT') {
                                 return Promise.resolve({
                                     ...next,
                                     urgency: 3,
```

Comment:
imo that would always reset the urgency, or could you change it?

Reply:
Good catch.